tasks/prelim.yml

```diff
@@ -3,10 +3,24 @@
 # List users in order to look files inside each home directory
 - name: "PRELIM | List users accounts"
   command: "awk -F: '{print $1}' /etc/passwd"
-  register: users
+  register: passwd
   changed_when: no
   check_mode: no
 
+- name: "PRELIM | List home directories"
+  find:
+      paths:
```

Comment:
maybe something like
`find /home -xdev -name .bash_profile -exec sh -c 'id -un $(basename $(dirname {})) 2> /dev/null' \;`

(not tested)

Reply:
Thanks, this is a good idea.
It would miss any user without a .bash_profile file, but that should be copied from /etc/skel when the user is created.
It also normalises the usernames, correcting case for instance.

I did have to add the `-maxdepth 2` option to `find` to avoid it picking up the users in my containers storage path.
`/home/user/.local/share/containers/storage/vfs/dir/xxx/root/.bash_profile`

I'll put another commit together when I'm able.